plugins/@grouparoo/facebook/__tests__/integration/export-records-custom.ts

```diff
@@ -55,33 +55,50 @@ function getUserList(count) {
 }
 
 async function getAudience(id) {
-  const audience = client.audience(id);
-  const fields = ["id", "name", "subtype", "approximate_count"];
-  const params = {};
-  await audience.get(fields, params);
-  return audience;
+  try {
+    const audience = client.audience(id);
+    const fields = ["id", "name", "subtype"]; // approximate_count removed?
+    const params = {};
+    await audience.get(fields, params);
+    return audience;
+  } catch (err) {
+    console.log(err);
```

Comment:
Why the extra logs throughout?

Reply:
Found it hard to debug stack and issue (generic Async stack and no error message) in the tests when it just threw the error, so I'd want these in the CI if they actually happen.